code/modules/projectiles/projectile.dm

```diff
@@ -228,11 +228,11 @@
 	qdel(src)
 
 //to get the correct limb (if any) for the projectile hit message
-/mob/living/proc/check_limb_hit(hit_zone)
+/mob/living/proc/check_hit_limb_name(hit_zone)
```

Comment:
Should this not be `check_hit_limb_zone`? It doesn't return the name of the limb.

Reply:
probably, when I get home I can rename it